assets/panel/debugger.properties

```diff
@@ -555,6 +555,9 @@ outline.header=Outline
 # LOCALIZATION NOTE (outline.noFunctions): Outline text when there are no functions to display
 outline.noFunctions=No functions
 
+# LOCALIZATION NOTE (outline.noFileSelected): Outline text when there are no functions to display
```

Comment:
The comment is wrong "when there are no functions to display" should be "Outline text when there are no files selected"